src/autoplex/data/rss/flows.py

```diff
@@ -68,14 +74,19 @@ def make(self):
         job_list = []
         final_structures = []
         for i, struct_number in enumerate(self.generated_struct_numbers):
+            cell_seed_path = None
             buildcell_option = None
-            if self.buildcell_options is not None:
+            if self.cell_seed_paths is not None:
+                assert len(self.generated_struct_numbers) == len(self.cell_seed_paths)
+                cell_seed_path = self.cell_seed_paths[i]
+            elif self.buildcell_options is not None:
```

Comment:
Better to raise an Error when both are none ? I am not that familiar with the buildcell program, but I think it would need either one of these parameters as input ?

Reply:
Nevermind ,I see defaults are set explicitly in https://github.com/YuanbinLiu/autoplex_pub/blob/f465858691bf96968a95448b0a6d362206194130/src/autoplex/data/rss/jobs.py#L26,